compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/res/DesktopSvgResources.desktop.kt

```diff
@@ -49,6 +49,7 @@ import org.jetbrains.skia.svg.SVGPreserveAspectRatioAlign
  * will be drawn with the specified size, density will have no effect.
  * @return the decoded SVG image associated with the resource
  */
+@Deprecated("Migrate to the compose resources library")
```

Comment:
Let's check how resources work in [InteliJ plugin integration](https://github.com/JetBrains/compose-multiplatform/tree/master/examples/intellij-plugin) before deprecation.

Reply:
commented below